examples/gen_test_data/gen_test_data/common.py

```diff
@@ -89,7 +90,10 @@ def print_progress(log_file_path: str, process):
 
     logger = get_logger("data.gen")
     logger.info(f"Click '{log_file_path}' to see detailed batch run log. Showing the progress here...")
-    log_pattern = re.compile(r".*execution.bulk\s+INFO\s+Finished (\d+) / (\d+) lines\.")
+    finished_log_pattern = re.compile(r".*execution.bulk\s+INFO\s+Finished (\d+) / (\d+) lines\.")
+    progress_log_pattern = re.compile(
+        r".*execution\.bulk\s+INFO\s+\[Lines]\s+\[Finished:\s+(\d+)]\s+\[Processing:\s+(\d+)]\s+\[Pending:\s+(\d+)]"
```

Comment:
Can we ignore `[Lines]` from regrex pattern? Maybe more robust.

Reply:
updated. please check again